### ISSUE TITLE
GET doesn't work

### ISSUE THREAD

**Issue by tberman** [skevy/graphiql-app#8]
When I switch over to GET, I receive:

```
TypeError: Failed to execute 'fetch' on 'Window': Request with GET/HEAD method cannot have body.
    at TypeError (native)
    at Object.graphQLFetcher [as fetcher] (file:///Users/todd.berman/Downloads/GraphiQL.app/Contents/Resources/app/dist/bundle.js:18:1115)
    at t._fetchQuery (file:///Users/todd.berman/Downloads/GraphiQL.app/Contents/Resources/app/dist/bundle.js:20:17540)
    at _runEditorQuery (file:///Users/todd.berman/Downloads/GraphiQL.app/Contents/Resources/app/dist/bundle.js:20:9601)
    at Object.u [as executeDispatch] (file:///Users/todd.berman/Downloads/GraphiQL.app/Contents/Resources/app/dist/bundle.js:1:7795)
    at C.executeDispatch (file:///Users/todd.berman/Downloads/GraphiQL.app/Contents/Resources/app/dist/bundle.js:17:25964)
    at a (file:///Users/todd.berman/Downloads/GraphiQL.app/Contents/Resources/app/dist/bundle.js:1:7727)
    at Object.s [as executeDispatchesInOrder] (file:///Users/todd.berman/Downloads/GraphiQL.app/Contents/Resources/app/dist/bundle.js:1:7848)
    at c (file:///Users/todd.berman/Downloads/GraphiQL.app/Contents/Resources/app/dist/bundle.js:2:22666)
    at Array.forEach (native)
```

in the response window.

**Comment by skevy** [comment]
I missed your PR - I'll take a look now.

**Comment by skevy** [comment]
Fixed by #13.
